attributes/network/11_ledger_tokens.adoc

```diff
@@ -0,0 +1,93 @@
+= Ledger Tokens Management Attribute (#11)
+:cddl: ./cddl/
+
+This attribute specifies that a ledger can have tokens managed by an authority.
+Without this attribute, tokens are defined at the genesis of the ledger, and cannot be created, minted, or burnt.
+
+Tokens described by this attribute are fungible and well specified.
+They would be equivalent to ERC-20 tokens.
+
+A server implementing this attribute SHOULD protect the endpoints described in this form in some way.
+For example, endpoints SHOULD error if the sender isn't from a certain address.
```

Comment:
How should the new symbol (Address) be generated?

Reply:
Clarified.